UnitsNet.Tests/CustomCode/InformationTests.cs

```diff
@@ -84,5 +84,17 @@ public void OneKBHas1000Bytes()
         {
             Assert.AreEqual(1000, Information.FromKilobytes(1).Bytes);
         }
+
+        [Test]
+        public void MaxValueIsCorrectForUnitWithBaseTypeDecimal()
+        {
+            Assert.AreEqual(decimal.MaxValue, Information.MaxValue.Bits);
```

Comment:
Maybe long wasn't sufficient to represent the largest types, from the top of my head I believe decimal has a wider range than long.